requirements.txt

```diff
@@ -11,3 +11,6 @@ typing==3.6.2
 beautifulsoup4==4.6.3
 GitPython==2.1.11
 python_json_logger==0.1.9
+git+https://github.com/ONSdigital/dp4py-config.git@master#egg=dp4py_config
```

Comment:
I'd maybe think about using pypi and pip these the usual way if you're doing custom libraries for prod. This works well enough, but enforces a git dependancy that doesn't need to be there.